src/main/java/seedu/heymatez/commons/util/StringUtil.java

```diff
@@ -49,20 +55,28 @@ public static String getDetails(Throwable t) {
     }
 
     /**
-     * Returns true if {@code s} represents a non-zero unsigned integer
+     * Returns an integer (1) VALID if {@code s} represents a non-zero unsigned integer or
      * e.g. 1, 2, 3, ..., {@code Integer.MAX_VALUE} <br>
-     * Will return false for any other non-null string input
+     * an integer (2) INVALID_INTEGER if {@code s} represents a non-positive integer.
+     * e.g. 0, -1, -2, ..., {@code Integer.MIN_VALUE} <br>
+     * Will return an integer (3) INVALID_INPUT for any other non-null string input
      * e.g. empty string, "-1", "0", "+1", and " 2 " (untrimmed), "3 0" (contains whitespace), "1 a" (contains letters)
      * @throws NullPointerException if {@code s} is null.
      */
-    public static boolean isNonZeroUnsignedInteger(String s) {
+    public static int isNonZeroUnsignedInteger(String s) {
```

Comment:
The method name should be modified too since the function no longer returns a boolean while 'isNonZeroUnsignedInteger' sounds like a boolean method name!

Reply:
i see... i will change it asap